### ISSUE TITLE
diff (staged) is empty

### ISSUE THREAD

**Issue by lynn2910** [jnsahaj/lumen#20]
Hey!

I've started to use `lumen`, and if it works perfectly, I have a strange error.
```shell
PS R:\IUT_Code\S3\24hmans> lumen draft
error: diff (staged) is empty
```

As you can see, it doesn't detect any changes. But when I run `git diff`:
```shell
PS R:\IUT_Code\S3\24hmans> git diff
diff --git a/frontend/src/views/HomeView.vue b/frontend/src/views/HomeView.vue
index c774958..8567b9c 100644
--- a/frontend/src/views/HomeView.vue
+++ b/frontend/src/views/HomeView.vue
@@ -1,112 +1,112 @@

...
```

Reloading the cache of git didn't worked.
Also, I found it pretty strange that the `lumen explain --diff` command works

**Comment by jnsahaj** [comment]
Hi, did you stage the changes? For example using `git add .`

**Comment by lynn2910** [comment]
Yes I did, it was one of the first things I checked. What's weird is that sometimes it works

**Comment by jnsahaj** [comment]
When you get this error, can you also check with `git diff --staged`

**Comment by jnsahaj** [comment]
Closing this for now since I don't believe this is a bug.
Feel free to open another issue if you encounter this again
